src/Object/Api/Mastodon/Token.php

```diff
@@ -22,7 +22,6 @@
 namespace Friendica\Object\Api\Mastodon;
 
 use Friendica\BaseDataTransferObject;
-use Friendica\Util\DateTimeFormat;
 
 /**
  * Class Error
```

Comment:
Wrong PHP doc for the class, please update the link to the documentation to https://docs.joinmastodon.org/entities/token/

Reply:
Details ...